lib/doorkeeper/orm/active_record/access_token.rb

```diff
@@ -10,6 +10,10 @@ def self.delete_all_for(application_id, resource_owner)
     end
     private_class_method :delete_all_for
 
+    def self.for(resource_owner)
```

Comment:
What's the opposite of `revoked`? Should this be called, for example, `active_for`, to make it explicit that we filter out revoked tokens?